docs/tutorial/docker_installation.md

```diff
@@ -190,3 +190,37 @@ collector could be organized. For another example how various components
 (as Docker images here) can work together, see a *Docker Compose*
 example with loosely coupled components
 [here](https://github.com/cybertec-postgresql/pgwatch/blob/master/docker-compose.yml).
+
+## Example of advanced setup using YAML files and dual sinks:
+
+pgwatch service in file `docker/docker-compose.yml` can look like this:
+```yaml
+  pgwatch:
+    image: cybertecpostgresql/pgwatch:latest
+    command:
+      - "--web-disable=true"
+      - "--sources=/sources.yaml"
+      - "--sink=postgresql://pgwatch@postgres:5432/pgwatch_metrics"
+      - "--sink=prometheus://:8080"
+    volumes:
+      - "./sources.yaml:/sources.yaml"
+    ports:
+      - "8080:8080"
+    depends_on:
+      postgres:
+        condition: service_healthy
+```
+
+Source file `sources.yaml` in the same directory:
+```yaml
+- name: demo
+  conn_str: postgresql://pgwatch:pgwatchadmin@postgres/pgwatch'
+  preset_metrics: exhaustive
+  is_enabled: true
+  group: default
+```
+
+Running this setup you get pgwatch that uses sources from YAML file and
+outputs measurements to postgres DB and exposes them for Prometheus 
+to scrape on port 8080 instead of WebUI (which is disabled by `--web-disable`).
+Metrics definition are built-in, you can examine definition in [`internal/metrics/metrics.yaml`](https://github.com/cybertec-postgresql/pgwatch/blob/master/internal/metrics/metrics.yaml).
```

Comment:
This is true, but I wanted to show here that you can provide your own metric definition file.

Reply:
Then we need to make one :) Otherwise people will just copy-paste and I want they know that metrics are always built-in

Reply:
And it's a bad style to link to `internal` folder in Go projects anyway

Reply:
All true, not only you are the lazy one 😁

Reply:
How about to add

Metric definition file `metrics.yaml` in the same directory will define only one metric `db_size` only for demonstration purposes. In real life you will have tens metrics easily.
```yaml
metrics:
    db_size:
        sqls:
            11: |-
                select /* pgwatch_generated */
                  (extract(epoch from now()) * 1e9)::int8 as epoch_ns,
                  pg_database_size(current_database()) as size_b,
                  (select sum(pg_total_relation_size(c.oid))::int8
                   from pg_class c join pg_namespace n on n.oid = c.relnamespace
                   where nspname = 'pg_catalog' and relkind = 'r'
                  ) as catalog_size_b
        gauges:
            - '*'
        statement_timeout_seconds: 300
```

Reply:
But we can skip this part now. I will add a special how-to chapter about custom metrics anyway

Reply:
Up to you 😁 You are the owner in the end, no?